recipes/scib/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "scib" %}
+{% set version = "1.1.4" %}
+
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x.x') }}
+
+source:
+  url: https://github.com/theislab/scib/archive/v{{ version }}.tar.gz
+  sha256: aebdad725b58a0bf8f3660a57e1ab81a06845632bb1b6cf9c64f6378d191154b
+  patches:
+    - setup_compiler.patch
+
+requirements:
+  host:
+    - python >=3.7,<3.12
+    - pip
+    - setuptools
+  build:
+    - {{ compiler('cxx') }}
+  run:
+    - anndata >=0.7.2
+    - deprecated
+    - h5py
+    - igraph >=0.10
+    - leidenalg
+    - llvmlite
+    - matplotlib-base
+    - numpy
+    - pandas
+    - pydot
+    - python >=3.7,<3.12
```

```suggestion
build:
  number: 0
  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
  skip: True  # [ py < 37 or py > 311]
  run_exports:
    - {{ pin_subpackage(name, max_pin='x.x') }}

source:
  url: https://github.com/theislab/scib/archive/v{{ version }}.tar.gz
  sha256: aebdad725b58a0bf8f3660a57e1ab81a06845632bb1b6cf9c64f6378d191154b
  patches:
    - setup_compiler.patch

requirements:
  host:
    - python
    - pip
    - setuptools
  build:
    - {{ compiler('cxx') }}
  run:
    - anndata >=0.7.2
    - deprecated
    - h5py
    - igraph >=0.10
    - leidenalg
    - llvmlite
    - matplotlib-base
    - numpy
    - pandas
    - pydot
    - python
```
Only artifacts for python `3.11` are built - is this intentional? If you would like the CI to have separate builds for each python `3.8` - `3.11`, please add the `skip` statement and remove the python version pinnings.

---

thank you for pointing this out, I was not aware of the correct way since this is my first pr

---

Not a problem, I have committed the suggested changes :)